docs/site/Using-strong-error-handler.md

```diff
@@ -0,0 +1,244 @@
+---
+title: 'Using strong-error-handler'
+lang: en
+keywords: LoopBack 4, error handling
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Using-strong-error-handler.html
+---
+
+# strong-error-handler
+
+This package is an error handler for use in both development (debug) and
+production environments.
+
+In LoopBack 4, the request handling process starts with the app's
+[sequence](https://loopback.io/doc/en/lb4/Sequence.html), a simple class with
+five injected helper methods in the constructor. It is the gatekeeper of all
+requests to the app. Errors are handled by one of the Sequence actions,
+[`reject`](Sequence.md#handling-errors), which calls `strong-error-handler`
+package.
+
+LB4 apps require 3.x versions of `strong-error-handler`.
+
+## Installation
+
+```bash
+$ npm install --save strong-error-handler
+```
+
+## Usage
+
+As mentioned above, errors are handled by the `reject` action of the app's
+[sequence](https://loopback.io/doc/en/lb4/Sequence.html). By default,
+implementation of `reject`, `strong-error-handler` omits details from error
+responses to prevent leaking sensitive information.
+
+`strong-error-handler` also can return full error stack traces and internal
+details of any error objects to the client in the HTTP responses once the
+`debug` flag is set. Please check
+[Sequence - Handling errors](Sequence.md#handling-errors) section for usages and
+examples.
+
+The module also exports `writeErrorToResponse`, a non-middleware flavor of the
+error handler:
+
+```js
+import {Server} from 'http';
+import {writeErrorToResponse} from 'strong-error-handler';
+const errHandlingOptions = {debug: true, log: true};
+
+http
```

Comment:
could you elaborate more about how to use/combine this in a lB4 application?

Reply:
I decide to remove this part as I think it's the same as the [Customizing log message] section below.